app/controllers/statistics_announcements_controller.rb

```diff
@@ -31,12 +21,4 @@ def expire_cache_for_index_on_next_announcement_expiry(announcements)
     time_to_releases = announcements.map {|ann| ann.release_date - Time.zone.now }.reject {|time_span| time_span <= 0 }
     expires_in((time_to_releases << Whitehall.default_cache_max_age).min)
   end
-
-  def expire_cache_for_show_on_release(announcement)
```

Comment:
Just to check my understand of publish intents is right: in this case, an intent is published (via a different `publishing-api` endpoint) "alongside" the current Statistics Announcement content item.  
When the path of the Statistics Announcement content item is fetched from `content-store`, the cache control config from the intent is merged into the content item.

The rendering app will then [respect the cache control config in the content item](https://github.com/alphagov/government-frontend/blob/master/app/controllers/content_items_controller.rb#L35-L390), reducing the TTL appropriately. And in the middle of that `gds-api-adaptors` will also respect the TTL in it's caching.

Have I got that right?